Add Docs For the AWS S3 Bucket ARN ignorelist (#163)
thinkst/canarytools-docs: docs/canarytokens/aws-s3-bucket-arn-ignorelist.md

        description: A valid flock_id
      - name: canarytoken
        required: true
        type: string
        description: A valid AWS S3 Bucket Canarytoken
    response: A JSON structure containing the ARN ignore list for the specified AWS S3 Bucket Canarytoken.
  save:
    name: Save AWS S3 Bucket Canarytoken ARN Ignore List
    url: /api/v1/canarytokens/s3bucket/ignorelist/save
    method: POST
    description: Save a new ARN ignore list for a specific AWS S3 Bucket Canarytoken.
    params:
      - name: auth_token
        required: true
        type: string
        description: A valid auth token
      - name: flock_id
        required: true
        type: string
        description: A valid flock_id
      - name: canarytoken
        required: true
        type: string
        description: A valid AWS S3 Bucket Canarytoken
      - name: ignores
        required: false
        type: string
        description: A space separated list of ARNs or ARN patterns.
    response: A JSON structure with result indicator.
---

# AWS S3 Bucket Canarytoken ARN ignore list

<APIEndpoints :endpoints="$page.frontmatter.endpoints" :path="$page.regularPath">

::::: slot details

> An AWS S3 Bucket Canarytoken monitors a S3 bucket of your choosing and generates alerts when that bucket is accessed.

Reviewer: ```suggestion
An AWS S3 Bucket Canarytoken monitors an S3 bucket of your choosing and generates an alert when that bucket is accessed.
```

        description: A valid flock_id
      - name: canarytoken
        required: true
        type: string
        description: A valid AWS S3 Bucket Canarytoken
    response: A JSON structure containing the ARN ignore list for the specified AWS S3 Bucket Canarytoken.
  save:
    name: Save AWS S3 Bucket Canarytoken ARN Ignore List
    url: /api/v1/canarytokens/s3bucket/ignorelist/save
    method: POST
    description: Save a new ARN ignore list for a specific AWS S3 Bucket Canarytoken.
    params:
      - name: auth_token
        required: true
        type: string
        description: A valid auth token
      - name: flock_id
        required: true
        type: string
        description: A valid flock_id
      - name: canarytoken
        required: true
        type: string
        description: A valid AWS S3 Bucket Canarytoken
      - name: ignores
        required: false
        type: string
        description: A space separated list of ARNs or ARN patterns.
    response: A JSON structure with result indicator.
---

# AWS S3 Bucket Canarytoken ARN ignore list

<APIEndpoints :endpoints="$page.frontmatter.endpoints" :path="$page.regularPath">

::::: slot details

> An AWS S3 Bucket Canarytoken monitors a S3 bucket of your choosing and generates alerts when that bucket is accessed.
> The same S3 bucket can be accessed by legitimate AWS resources that can be identified with an Amazon Resource Name (ARN).
> 
> For the AWS S3 Bucket Canarytoken to not generate an alert when a legitimate AWS resource accesses your chosen bucket,
> you must add that resource's ARN to the ARN ignorelist of an AWS S3 Bucket Canarytoken.

Reviewer 2: ```suggestion
An AWS S3 Bucket Canarytoken monitors a S3 bucket of your choosing and generates alerts when that bucket is accessed. 

The  Amazon Resource Names (ARN's) of AWS resources that need to access the bucket legitimately can be added to an ignore list and won't generate alerts when they access the bucket.  
```
